tests/test_requirements.py

```diff
@@ -0,0 +1,192 @@
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import pytest
+import yaml
+from pkg_resources import Requirement
+
+REQUIREMENTS_FILES = [
```

Comment:
Should requirements.win64.conda.yml be part of the picture of inclusions and constraints?

Reply:
Thanks for catching this, I've added this in [f195d96](https://github.com/TissueImageAnalytics/tiatoolbox/pull/392/commits/f195d96e64a4542ca5543898f3637bd947c1855b)